pkgs/development/libraries/libxml2/default.nix

```diff
@@ -47,8 +48,6 @@ stdenv.mkDerivation rec {
     ./utf8-xmlErrorFuncHandler.patch
   ];
 
-  strictDeps = true;
```

Comment:
Maybe this can now be added back? Also maybe it looks for the python in nativeBuildInputs?

Reply:
No, that still does not work.